packages/store/README.md

```diff
@@ -0,0 +1,143 @@
+# @scalar/store
+
+A powerful server-side data store for managing OpenAPI documents. This package provides a flexible solution for handling multiple OpenAPI specifications in a workspace environment, with support for both SSR (Server-Side Rendering) and static modes.
+
+## Usage
+
+Create a new store in ssr mode
+
+```ts
+// Create the store
+const store = createServerWorkspace({
+  baseUrl: 'example.com',
+  mode: 'ssr',
+  meta: { 'x-scalar-active-document': 'document-name' },
+  documents: [
+    {
+      name: 'document-name',
+      meta: {},
+      document: {
+        openapi: '3.1.1',
+        info: {
+          title: 'Hello World',
+          version: '1.0.0',
+        },
+        components: {
+          schemas: {
+            Person: {
+              type: 'object',
+              properties: {
+                name: { type: 'string' },
+              },
+            },
+            User: {
+              $ref: '#/components/schemas/Person',
+            },
+          },
+        },
+      },
+    },
+  ],
+})
+
+// Add a new document to the store
+store.addDocument({
+  openapi: '3.1.1',
+  info: {
+    title: 'Hello World',
+    version: '1.0.0',
+  },
+  components: {
+    schemas: {
+      Person: {
+        type: 'object',
+        properties: {
+          name: { type: 'string' },
+        },
+      },
+      User: {
+        $ref: '#/components/schemas/Person',
+      },
+    },
+  },
+}, {
+  name: 'document-2',
+  "x-scalar-active-server": "server1"
+})
+
+// Get the workspace
+// Workspace is going to keep all the sparse documents
+const workspace = store.getWorkspace()
+
+// Get chucks using json pointers
+const chunk = store.get('#/document-name/components/schemas/Person')
```

Comment:
for someone working with OpenAPI documents all the time (me), it’s super strange to see this `#/document-name` prefix. should we do store.get('my-document', '#/components/schemas/foobar')`?